x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -220,6 +223,38 @@ describe('when on the list page', () => {
         expect(firstPolicyName.getAttribute('href')).toContain(`policy/${firstPolicyID}`);
       });
 
+      it('should poll', async () => {
```

Comment:
this test is loooonnng and does a few assertions.
We should consider in the future perhaps making `polling/autorefresh` a dedicated `describe()` block that sets up the needed mocks/conditions and then help break this up into multiple smaller tests

Reply:
I can do that here, makes sense